Yoast/wordpress-seo
# Fixed How To json/ld formatting and added unit-test

## Summary

This PR can be summarized in the following changelog entry:

* Restructured How To block's structured data to meet Google's guidelines.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In Gutenberg, create a new post with a How To block.
* Add a how to step with a title and description.
* Add a how to step with a title, description and an image.
* Publish the post and inspect the `application/ld-json` script element.
* Optionally copy the content of the `script` element into https://jsonformatter.curiousconcept.com/ for better readability.
* Make sure all changes described in #11001 have been applied.
* Also copy the contents into https://search.google.com/structured-data/testing-tool/ and make sure the validation passes.

## Quality assurance


Fixes #11001
